test/include/util/storage_test_util.h

```diff
@@ -64,6 +64,11 @@ struct StorageTestUtil {
     return reinterpret_cast<A *>(reinterpret_cast<byte *>(ptr) + bytes);
   }
 
+  // Align some bytes that is aligned to 8 bytes
```

Comment:
"Allocate some bytes"?

Reply:
I'll just move this into `common` and make it the wrapper around `posix_memalign`